build.sh

```diff
@@ -280,6 +286,11 @@ fi
 source $scriptroot/eng/common/native/init-os-and-arch.sh
 source $scriptroot/eng/common/native/init-distro-rid.sh
 initDistroRidGlobal "$os" "$arch" ""
+if [[ -n "$target_rid" ]]; then
+  # override with user's preferred RID
+  export __DistroRid="$target_rid"
```

Comment:
```suggestion
```

I think we can leave this out.

Reply:
That affects OutputRID calculation in Directory.Build.props.

Reply:
I'm not sure what you mean. For runtime, we're passing the rid down here:

https://github.com/dotnet/runtime/blob/40787436cba1b732a2507aaba2cd2a1e6f809a50/eng/DotNetBuild.props#L58

If there are places where `__DistroRid` overrides `TargetRid` then we should fix them.